src/main/java/embl/ebi/variation/eva/pipeline/jobs/CommonJobStepInitialization.java

```diff
@@ -0,0 +1,45 @@
+package embl.ebi.variation.eva.pipeline.jobs;
+
+import embl.ebi.variation.eva.VariantJobsArgs;
+import org.opencb.datastore.core.ObjectMap;
+import org.springframework.batch.core.Step;
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory;
+import org.springframework.batch.core.step.builder.StepBuilder;
+import org.springframework.batch.core.step.builder.TaskletStepBuilder;
+import org.springframework.batch.core.step.tasklet.Tasklet;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Import;
+
+@Import({VariantJobsArgs.class})
+public class CommonJobStepInitialization {
+
+    @Autowired
+    private VariantJobsArgs variantJobsArgs;
+
+    @Autowired
+    private StepBuilderFactory stepBuilderFactory;
+
+    /**
+     * Initialize a Step with common configuration
+     * @param tasklet to be initialized with common configuration
+     */
+    protected void initStep(final TaskletStepBuilder tasklet) {
```

Comment:
This method seems to be used only from `generateStep` now. Could there be any benefit on keeping them separate?